source/TimePicker.js

```diff
@@ -554,10 +594,10 @@
 		hourPickerChanged: function (sender, event) {
 			if(this.syncingPickers) return true;
 
-			var hour = event.value;
+			var hour = this.value.getHours() + (event.originator.get('offset'));
```

Comment:
Wonder if we should document the `offset` property of `HourPicker` and/ or just use direct property access i.e. `event.originator.offset`.